Content/Items/BuriedArtifacts/Artifacts.CommonDesertArtifactItems.cs

```diff
@@ -0,0 +1,42 @@
+﻿using StarlightRiver.Core;
+using Terraria;
+
+namespace StarlightRiver.Content.Items.BuriedArtifacts
+{
+    public class DesertArtifact1Item : QuickMaterial
```

Comment:
These should probably get autoloaded with the artifacts isntead of being defined seperately -- this could probably included in that parameterized loading method

Reply:
Except they're going to all have unique/special behaviors in the future